### PR TITLE
Redesigned the website of Soundnode

### PR DESCRIPTION
**Pull request by ghost**
I redesigned the website of Soundnode. Please let me know what you think about it.

Screenshot

### CODE REVIEW

**Comment by jakejarrett** [comment]
@wb9688 The website will throw exception errors every time i try to view the page after running npm install

```
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: not implemented
    at Readable._read (_stream_readable.js:464:22)
    at Readable.read (_stream_readable.js:341:10)
    at resume_ (_stream_readable.js:725:12)
    at _stream_readable.js:717:7
    at process._tickCallback (node.js:355:11)
```

edit- Some tweaks that i think would make it look a lot more polished-

For navigation-
Set background to #333333
Set color to #EEEEEE
Set border-color to rgba(0, 0, 0, 0.1)
(and any other tweaks that'd require a dark navigation instead of light)

For Buttons-
Remove inset box-shadow (Including on hover)
Remove border (Including on hover)

here is a version with the dark navigation
screenshot from 2016-04-30 16-14-59

**Comment by ghost** [comment]
I don't know why it isn't working for you. What Node.js version are you
using? By me it works on Windows 10 and Lubuntu 16.04 both with Node.js
v5.11.0.

**Comment by ghost** [comment]
@jakejarrett I had a purple header in #6, but @weblancaster wanted it to be white. So, should I've a black, white or purple header? Btw I'll change the buttons.

**Comment by weblancaster** [comment]
Cool, let us know when you have the last version to show to us. thanks.

**Comment by ghost** [comment]
@weblancaster @jakejarrett should I've a black, white or purple header? When I've got answer on that question I'm gonna add the changes.

**Comment by weblancaster** [comment]
I think black looks good.. that doesn't really matter though we can decide the color once the structure is the same and the files are correct

**Comment by ghost** [comment]
But for changing the color I've to rebuild Bootstrap. Btw I'll change it
tomorrow (I've no time today).

**Comment by ghost** [comment]
@weblancaster @jakejarrett I've updated it:

Screenshot

**Comment by weblancaster** [comment]
looks good @wb9688 now we just need to clean up the commit because there is too much changes that we don't need.

Do you need bootstrap? can you remove the zip from the commit? can we keep the same file structure we had before? can we not use less or sass?

**Comment by ghost** [comment]
Yes, I need Bootstrap, but it's just a customized and purified build so it
makes no big difference. Which zip do you mean? What things should I change
in the file structure (or can you change it)? I don't use less or sass.

File: server.js [hunk 1]
```diff
@@ -1,24 +1,13 @@
...
-router.get('/', function(req, res, next) {
-    res.render('index.html');
-});
-
-app.use('/', router);
-
-app.listen(port);
-console.log('App running on port', port);
\ No newline at end of file
+var express = require('express'),
```
Comment (weblancaster) [inline server.js:1]: the server should stay as it was.. why did you remove bodyParser? methodOverride? what is the benefit of having shrink-ray over compression?

Leave the server rendering html that way will help with SEO

**Comment by weblancaster** [comment]
Everything inside `public/build` and `public/downloads` should not be part of the commit

**Comment by ghost** [comment]
I'll change server.js back, but with a few changes:
- Using `shrink-ray` instead of `compression`, because it's faster
- Removing the `app.get` statement, because it's useless as you're already
  serving `index.html` through `express.static`
- Executing `console.log` through `app.listen`

And I'd change everything in `public/(build|downloads)` back to before and
create a new PR.
